doc/commands/module.8.rst

```diff
@@ -46,31 +48,57 @@ Subcommands
 ``info``
     | Print details about module streams.
 
-``provides``
-    | Print module and module profile the specified packages come from.
-
 ``enable``
     | Enable module streams and make their packages available.
 
 ``disable``
     | Disable modules including all their streams.
 
-``switch-to``
-    | Enable different module streams, upgrade their profiles and distro-sync packages.
-
 ``reset``
     | Reset module state so it's no longer enabled or disabled.
 
-``install``
-    | Install module profiles, including their packages.
 
-``remove``
-    | Remove installed module profiles including their packages.
 
+``--skip-broken``
+    | Used with ``enable`` command to resolve any dependency problems by removing items that are causing problems from the transaction.
 
-Options
+``--skip-unavailable``
+    | Used with ``enable``, ``disable`` and ``reset`` commands to allow skipping modules that are not available in repositories.
+    | All remaining items will be processed.
 =======
+Options for ``list`` and ``info``
+=================================
```

Comment:
The overline (`=======`) probably shouldn't be there. It results in "Title overline & underline mismatch." error when building the documentation.

Reply:
Oh, and the `--skip-broken` and `--skip-unavailable` options are both in the "Subcommands" section and the "Options for ``enable``, ``disable``, ``reset``" section now. Looks like it's all a result of a merge conflict resolving?

Reply:
Right, some oversight during merging. Should be OK now.